.github/actions/build_micro_service_ko/action.yml

```
@@ -0,0 +1,78 @@
+name: Build micro service (KO)
+description: Build and upload a container image for a Constellation micro-service
+inputs:
+  name:
+    description: "Name of the micro-service"
+    required: true
+  koConfig:
+    description: "Path to the .ko.yaml config file"
+    default: ".ko.yaml"
+    required: false
+  pseudoVersion:
+    description: "Check if pseudo-version should be generated"
+    default: "false"
+    required: true
+  koTarget:
+    description: "Go package to build with ko"
+    required: true
+  pushTag:
+    description: "Use this image tag"
+    required: false
+  githubToken:
+    description: "GitHub authorization token"
+    required: true
+  generateKoSBOM:
+    description: "Generate unsigned ko SBOM"
+    required: false
+    default: "false"
+  cosignPublicKey:
+    description: "Cosign public key"
+    required: false
+  cosignPrivateKey:
+    description: "Cosign private key"
+    required: false
+  cosignPassword:
+    description: "Password for Cosign private key"
+    required: false
+
+# Linux runner only
+runs:
+  using: "composite"
+  steps:
+    - name: Determine pseudo version
+      #if: ${{ inputs.pseudoVersion == 'true' }}
+      if: ${{ failure() }}
```

Probably left over debugging code? 

```suggestion
      if: ${{ inputs.pseudoVersion == 'true' }}
```

---

Yes, this is a debug relic and should be removed.